packages/slice-machine/lib/builders/common/Zone/Card/components/NewField.js

```diff
@@ -131,10 +197,10 @@ const NewField = ({ widgetTypeName, fields, onSave, onCancelNewField }) => {
                     },
                   }}
                 />
-                <ErrorTooltip errors={errors} />
+                <ErrorTooltip error={errors.id} />
```

Comment:
the ErrorTooltip was just taking the errors and rendering the `.id` in it. I have moved this out since now the ErrorTooltip may have to render the `errors.id` or the `errors.label` (in each of the fields)

Reply:
neat :)